console/src/components/Services/Data/TableBrowseRows/ViewActions.js

```diff
@@ -89,13 +86,12 @@ const vMakeRequest = () => {
     };
     return dispatch(requestAction(url, options)).then(
       data => {
-        const currentTable = getState().tables.currentTable;
-        if (originalTable === currentTable) {
+        if (originalTable === tables.currentTable) {
```

Comment:
This will always be true?

Reply:
Nope. currentTable is derived from state again in the fn and can be different as it async because it is in `then()` of requestAction.

Reply:
Sorry. this seems like another instance of the issue I fixed